crates/edr_provider/src/lib.rs

```diff
@@ -482,6 +482,15 @@ impl<LoggerErrorT: Debug + Send + Sync + 'static, TimerT: Clone + TimeSinceEpoch
     }
 }
 
+#[cfg(feature = "test-utils")]
+impl<LoggerErrorT: Debug + Send + Sync + 'static, TimerT: Clone + TimeSinceEpoch>
+    Provider<LoggerErrorT, TimerT>
+{
+    pub fn data_mut(&self) -> tokio::sync::MutexGuard<'_, ProviderData<LoggerErrorT, TimerT>> {
```

Comment:
This doesn't seem to be used.

Reply:
Good catch. This was leftover from a prior iteration. Resolved in https://github.com/NomicFoundation/edr/pull/805/commits/e41b0a759e8aee62d28366de5588c5e1d11d33e1